.github/workflows/build_binaries_on_new_releases.yml

```diff
@@ -24,8 +23,11 @@ jobs:
           os: ubuntu-latest
           target: arm-unknown-linux-gnueabihf
         - name: macos
-          os: macos-latest
+          os: macos-13
```

Comment:
- macos-13: amd64
- macos-14: arm64

Reply:
I confirmed that macos-13 with arm64 and macos-14 with amd64 require larger runners.
https://github.com/actions/runner-images/tree/main?tab=readme-ov-file#available-images